simple_ldap_user/simple_ldap_user.admin.inc

```diff
@@ -295,10 +295,18 @@ function simple_ldap_user_import($form, &$form_state) {
   asort($users);
 
   // Only selects new users
-  $drupal_users = entity_load('user');
+  
   $new_names = $users;
-  foreach ($drupal_users as $user) {
-      unset($new_names[strtoupper($user->name)]);
+  $sql="select name from {users} where name= :name";
+  foreach($new_names as $item){
+     $result=db_query($sql, array(':name'=>$item));
```

Comment:
I was really hoping that the db_query() call could prepare the `PDOStatement` outside of the foreach, with calls to assign the parameter and execute the statement within the foreach.  We didn't see a way to do this within Drupal, however.